Changes some defaults in recent activity defaults.
1. by default developer selected is YOU
as it's easier to select All from list (it's always First), than You.
2. change default hours 24h

--- eventum-20060913/reports/recent_activity.php~	2006-09-13 21:08:24.384131339 +0300
+++ eventum-20060913/reports/recent_activity.php	2006-09-13 23:42:06.870834513 +0300
@@ -64,6 +64,7 @@
 $tpl->assign(array(
     "units" =>  $units,
     "users" => Project::getUserAssocList($prj_id, 'active', User::getRoleID('Customer')),
+    "developer" => $usr_id,
     "type_list" =>  $type_list,
     "activity_types"    =>  $_REQUEST['activity_types']
 ));
--- eventum-20060913/templates/reports/recent_activity.tpl.html~	2006-09-13 21:08:24.214127523 +0300
+++ eventum-20060913/templates/reports/recent_activity.tpl.html	2006-09-13 23:42:06.950836304 +0300
@@ -42,7 +42,7 @@
             <b>{t}Activity in Past{/t}:</b>
           </td>
           <td width="200">
-            <input type="text" size="3" name="amount" value="{$amount|default:36}" class="default" style="text-align: right">&nbsp;
+            <input type="text" size="3" name="amount" value="{$amount|default:24}" class="default" style="text-align: right">&nbsp;
             <select name="unit" class="default">
                 {html_options options=$units selected=$unit}
             </select>
